src/raymath.h

```diff
@@ -154,6 +154,17 @@ RMDEF float Lerp(float start, float end, float amount)
     return start + amount*(end - start);
 }
 
+// Normalize input value within input range
+RMDEF float Norm(float value, float start, float end)
+{
+    return (value - start) / (end - start);
+}
+
+// Remap input value within input range to output range
+RMDEF float Remap(float value, float inputStart, float inputEnd, float outputStart, float outputEnd) {
```

Comment:
Also, is `Remap()` function name standard? Do other math libraries use this naming convention?

Reply:
I'm pretty sure they'll never included this function because if they have already norm and lerp. But this name is the most used I believe. The second common name is just "map".